pandas/tests/indexes/interval/test_interval.py

```diff
@@ -924,6 +926,16 @@ def test_interval_index_error_and_warning(self):
         with tm.assert_produces_warning(FutureWarning, match=msg_warn):
             IntervalIndex.from_tuples([(0, 1), (0.5, 1.5)], closed="both")
 
+    @pytest.mark.parametrize("box", [IntervalIndex, array, list])
+    def test_get_indexer_interval_index(self, box):
```

Comment:
Could you move this test to `pandas/tests/indexes/interval/test_indexing.py`?

Reply:
Hi @mroeschke sorry I goofed up!
Moved now.